docs/cvl/functions.md

```diff
@@ -43,3 +43,77 @@ Using CVL functions
 -------------------
   CVL Function may be called from within a rule, or from within another CVL function.
 
+Math and rounding summaries
+---------------------------
+
+In real protocols, arithmetic often requires precise rounding envelopes. CVL functions are a clean way to centralize these rules as small, reusable summaries.
+
+- Round-aware `mulDiv` abstraction with explicit direction:
+
+  ```cvl
+  // Rounds up or down depending on a Math.Rounding enum
+  function mulDivCVL(uint256 x, uint256 y, uint256 denominator, Math.Rounding rounding) returns uint256 {
```

Comment:
incomplete. Math is defined in OpenZeppelin's Math.sol, so if it is somehow not part of the project we work on, the `Math.Rounding` won't type check